src/instruction.cpp

```diff
@@ -64,10 +64,8 @@ void instruction::replace(const shape& r)
         result = r;
         for(auto&& ins : output)
         {
-            if(ins->name() == "@return")
-                continue;
-
-            assert(ins->name().front() != '@');
+            if(ins->name() != "@return")
+                assert(ins->name().front() != '@');
```

Comment:
Instead write this like `assert(ins->name() == "@return" or ins->name().front() != '@')`.

Reply:
Changed. Thanks.